webrtc-stats.html

```diff
@@ -1988,40 +1591,27 @@ <h3>
              DOMString            senderId;
              DOMString            remoteId;
              DOMString            rid;
-             DOMHighResTimeStamp  lastPacketSentTimestamp;
              unsigned long long   headerBytesSent;
-             unsigned long        packetsDiscardedOnSend;
```

Comment:
Revive this (packetsDiscardedOnSend)

Reply:
Correction: Don't revive this on the outbound-rtp... it does exist on candidate-pair though